knimin/lib/qiita_jira_util.py

```diff
@@ -72,15 +195,266 @@ def create_study(title, abstract, description, alias, qiita_user,
+        study = db.read_study(study_id)
+        issue_key = '%s-4' % study['jira_id']
+        jira_handler.add_comment(
+            issue_key, "Samples have been plated")
+
+    jira_handler
+    return dna_plates
+
+
+def prepare_targeted_libraries(plate_links, email, robot, tm300tool,
+                               tm50tool, mastermix_lot, water_lot):
+    """Stores the targeted plate library information
+
+    Parameters
+    ----------
+    plate_links : list of dicts
+        A list of {'dna_plate_id': int, 'primer_plate_id': int} linking
+        a DNA plate with the primer plate used
+    email : str
+        The email of the user doing the library prep
+    robot : str
+        The name of the robot used
+    tm300tool : str
+        The name of the TM300-8 tool used
+    tm50tool : str
+        The name of the TM50-8 tool used
+    mastermix_lot : str
+        The mastermix lot used
+    water_lot : str
+        The water lot used
+
+    Returns
+    -------
+    list of int
+        The new target_plate ids created
+    """
+    targeted_plate_ids = db.prepare_targeted_libraries(
+        plate_links, email, robot, tm300tool, tm50tool,
+        mastermix_lot, water_lot)
+
+    studies = []
+    for val in plate_links:
+        sample_plate = db.read_sample_plate(
+            db.read_dna_plate(val['dna_plate_id'])['sample_plate_id'])
+
+        studies.extend(sample_plate['studies'])
+
+    for study_id in set(studies):
+        study = db.read_study(study_id)
+        issue_key = '%s-4' % study['jira_id']
+        jira_handler.add_comment(
+            issue_key, "Target gene libraries have been prepared")
+
+    return targeted_plate_ids
+
+
+def create_sequencing_run(pool_id, email, sequencer, reagent_type,
+                          reagent_lot):
+    """Stores the sequencing run information
+
+    Parameters
+    ----------
+    pool_id : int
+        The pool being sequenced
+    email : str
+        The email of the user preparing the run
+    sequencer : id
+        The sequencer id
+    reagent_type : str
+        The reagent type
+    reagent_lot : str
+        The reagent lot
+
+    Returns
+    -------
+    int
+        The run id
+    """
+    run_id = db.create_sequencing_run(pool_id, email, sequencer, reagent_type,
+                                      reagent_lot)
+    studies = []
+    for targeted_pool in db.read_pool(pool_id)['targeted_pools']:
+        targeted_plate = db.read_targeted_plate(
+            targeted_pool['targeted_plate_id'])
+        dna_plate = db.read_dna_plate(targeted_plate['dna_plate_id'])
+        sample_plate = db.read_sample_plate(dna_plate['sample_plate_id'])
+        studies.extend(sample_plate['studies'])
+
+    for study_id in set(studies):
+        study = db.read_study(study_id)
+        issue_key = '%s-4' % study['jira_id']
+        jira_handler.add_comment(
+            issue_key, "Pools have been sent for sequencing")
+
+    return run_id
+
+
+ISSUE1_DESC = """
```

Comment:
Can comments be placed above each of these descriptions for what issue they correspond to? eg:

```python
# 1 - Project initiation
ISSUE1_DESC = """ ..."""
```